sos/report/plugins/system.py

```diff
@@ -23,6 +23,14 @@ def setup(self):
             "/etc/sysconfig",
             "/etc/default",
             "/etc/environment",
+            "info -o vd",
+            "info -o pd",
+            "info -o hba",
+            "arcconf getconfig 1",
```

Comment:
I'm not familiar with `arcconf`, but a quick search shows it is for adaptec RAID controllers. It does not belong in `system`.